js_modules/dagster-ui/packages/ui-core/src/overview/OverviewTimelineRoot.tsx

```diff
@@ -14,7 +14,7 @@ import {WorkspaceContext} from '../workspace/WorkspaceContext';
 import {buildRepoAddress} from '../workspace/buildRepoAddress';
 const LOOKAHEAD_HOURS = 1;
 const ONE_HOUR = 60 * 60 * 1000;
-const POLL_INTERVAL = 60 * 1000;
+const POLL_INTERVAL = 5 * 1000;
```

Comment:
Lowered the poll interval since we only fetch for the missing ranges.
Increased the poll interview for FutureTicks / Ongoing runs below.